### ISSUE TITLE
Impersonate user account

### ISSUE THREAD

**Issue by cturbelin** [influenzanet/user-management-service#3]
Just an idea: 
Sometimes users report a problem that is very difficult to reproduce or understand technically, to help them it could be interesting the user can allow the team to use their account in a secure way:

User can give a (very) temporary authorization to impersonate their account and test things.

Well, I'm not sure it's a *good* idea but it can be helpful. Otherwise teamviewer can do the job...

**Comment by phev8** [comment]
Do you have an example, what is a typical issue when user would want this?
Maybe there are better ways (e.g. user can send local debug log files - using a dedicated button in the app/webapp, or record a session in the client) if it's to fix an error. Or is it rather a UX issue, where specific user groups are not familiar with some user interface concepts, and therefor need support to achieve something with the platform?

**Comment by cturbelin** [comment]
Yes, for example to test the survey rules. As rules can depends on complex data it could be very difficult to understand what is the problem cause. The user only has the result : a question is showing (sometimes in a wrong situation) or not. It's very difficult to reproduce the problem without all the user's context (e.g. data).
We also had this issue with personalized results : only the user's context helps to understand the problem, and it's not 

May be it's just for the survey. For simpler problem yes the description of the issue by the user should be  enough. 
Actually, maybe it's not impersonate fully the account, but for example in "dev" mode, to be able to show the interface using data from a specific account (using participant anonymized UID for ex)..
